src/types/rule.ts

```diff
@@ -26,6 +26,9 @@ export interface IRule<T = string, O = any> {
   // Filter the target down to a subset[] with a JSON path
   given: string;
 
+  // If true, rule will operate on original (unresolved) data
+  unresolved?: boolean;
```

Comment:
Would it be better to swap? `unresolved: false` sounds a bit awkward and double-negative. Would something like `isResolved: true` be easier for others to read and reason about?